pctiler/pctiler/reader.py

```diff
@@ -137,50 +90,6 @@ def assets_for_tile(
         bbox = self.tms.bounds(morecantile.Tile(x, y, z))
         return self.get_assets(Polygon.from_bounds(*bbox), **kwargs)
 
-    @cached(
-        cache=TTLCache(maxsize=cache_config.maxsize, ttl=cache_config.ttl),
-        key=get_cache_key,
-    )
-    def get_assets(
-        self,
-        geom: Union[Point, Polygon],
-        fields: Optional[Dict[str, Any]] = None,
-        scan_limit: Optional[int] = None,
-        items_limit: Optional[int] = None,
-        time_limit: Optional[int] = None,
-        exitwhenfull: Optional[bool] = None,
-        skipcovered: Optional[bool] = None,
-    ) -> List[Dict]:
-        """Find assets."""
-        fields = fields or {
-            "include": ["assets", "id", "bbox", "collection"],
```

Comment:
`"collection"` is now also in titiler-pgstac https://github.com/stac-utils/titiler-pgstac/blob/84134d5ec6119e03d89560249858db3ccb1b42ea/titiler/pgstac/mosaic.py#L196-L198